libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -437,6 +448,78 @@ export const m = defineMessages({
   },
 })
 
+export const employerForm = defineMessages({
+  formTitle: {
+    id: 'pl.application:form.title',
+    defaultMessage: 'Samþykki vinnuveitanda fyrir umsókn um foreldraorlof',
+    description: 'Employer approval for parental leave application',
+  },
+  reviewSection: {
+    id: 'pl.application:review.section',
+    defaultMessage: 'Samþykki vinnuveitanda',
+    description: 'Employer approval',
+  },
+  reviewMultiTitle: {
+    id: 'pl.application:review.multi.title',
+    defaultMessage:
+      'Starfsmaður þinn hefur sótt um foreldraorlof. Samþykkir þú valin tímabil hans?',
```

Comment:
Same as above with foreldraorlof. Also, `Samþykkir þú valin tímabil hans?` should be `Samþykkir þú valið tímabil hans?` or better yet, `Samþykkir þú tímabilið sem hann valdi?`